woocommerce/woocommerce-android
# [Woo POS] Implement skeleton of the i3 of the wireframes

Closes: #11644


### Description

The PR brings a sceleton for the future "home" screen for the POS:
* Includes communication between top/down levels
* Home screen that contains 3 subscreens: Products, Cart, Totals

### Steps to reproduce


### Testing information

* Click checkout and products button
* Try "back" click
 Back click from the products screen is not implemented as this should call "exit POS" functionality which maybe a dialog or not sure what

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/de995e72-7abc-461a-8f19-e8a89127483f
